src/EasyCaching.HybridCache/HybridCachingProvider.cs

```diff
@@ -203,6 +205,7 @@ public async Task<bool> ExistsAsync(string cacheKey)
         /// </summary>
         /// <returns>The get.</returns>
         /// <param name="cacheKey">Cache key.</param>
+        /// <param name="cancellationToken">CancellationToken</param>
```

Comment:
XML comment has a param tag for 'cancellationToken', but there is no parameter by that name